marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/EnvironmentController.java

```diff
@@ -18,145 +18,94 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
-import com.marklogic.appdeployer.AppConfig;
-import com.marklogic.hub.HubProject;
-import com.marklogic.hub.central.exceptions.BadRequestException;
-import com.marklogic.hub.central.exceptions.ProjectDirectoryException;
-import com.marklogic.hub.central.listener.UIDeployListener;
-import com.marklogic.hub.central.models.HubConfigSession;
-import com.marklogic.hub.central.services.DataHubProjectUtils;
-import com.marklogic.hub.central.services.DataHubService;
-import com.marklogic.hub.central.services.EnvironmentService;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import org.apache.commons.lang3.StringUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import com.marklogic.client.ext.helper.LoggingObject;
+import com.marklogic.hub.central.HubCentral;
+import com.marklogic.hub.impl.Versions;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.MediaType;
-import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.core.env.Environment;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.multipart.MultipartFile;
 
-@Controller
-@RequestMapping("/api/environment")
-public class EnvironmentController {
-    protected final Logger logger = LoggerFactory.getLogger(this.getClass());
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashMap;
+import java.util.Map;
 
-    @Autowired
-    EnvironmentService environmentService;
+@Controller
+public class EnvironmentController extends LoggingObject {
 
     @Autowired
-    HubConfigSession hubConfig;
+    HubCentral hubCentral;
 
     @Autowired
-    DataHubService dataHubService;
+    Versions versions;
 
     @Autowired
-    private SimpMessagingTemplate template;
+    Environment environment;
 
     private ObjectMapper mapper = new ObjectMapper();
 
-    @RequestMapping(value = "/initialized", method = RequestMethod.GET)
+    @RequestMapping(method = RequestMethod.GET)
```

Comment:
We still need the request path to be
```suggestion
    @RequestMapping(value = "/api/info", method = RequestMethod.GET)
```

Reply:
Thanks, forgot that when copying the method over.